### PR TITLE
Feature integration samples

### PR DESCRIPTION
**Pull request by vijikarthi**
Added basic sanity test routine to test Pravega reader/writer in Nautilus environment. We can add more test stubs to this module to cover additional tests (negative/scaling etc.,)

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -4,7 +4,7 @@ projectVersion=0.1.0-SNAPSHOT
 # Dependency versions
 pravegaVersion=0.1.0-SNAPSHOT
 pravegaConnectorsVersion=0.1.0-SNAPSHOT
-flinkVersion=1.3-SNAPSHOT
+flinkVersion=1.3.1
```
Comment (chrisdail) [inline gradle.properties]: We may want to hold this until after EAP releases. 1.3.1 is only on master.

File: sanity-samples/src/main/java/io/pravega/samples/flink/StreamUtils.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import io.pravega.client.ClientFactory;
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.net.URI;
+import java.nio.ByteBuffer;
+
+public class StreamUtils {
```
Comment (chrisdail) [inline sanity-samples/src/main/java/io/pravega/samples/flink/StreamUtils.java]: See https://github.com/pravega/flink-connectors/blob/master/src/main/java/io/pravega/connectors/flink/util/FlinkPravegaParams.java
We should be using that for nautilus. It has helpers for creating streams and readers/writers.

File: sanity-samples/src/main/java/io/pravega/samples/flink/testruns/ExactlyOnceWriterTest.java [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+	public static void main(String[] args) {
+
+		log.info("Starting ExactlyOnceWriterTest Main...");
+
+		ParameterTool params = ParameterTool.fromArgs(args);
+		log.info("Parameter Tool: {}", params.toMap());
+
+		Parameters parameters = new Parameters();
+		parameters.controller =  params.get("controller", StreamUtils.DEFAULT_CONTROLLER_URI);
```
Comment (chrisdail) [inline sanity-samples/src/main/java/io/pravega/samples/flink/testruns/ExactlyOnceWriterTest.java]: Should use FlinkPravegaParams

File: gradle.properties (hunk 1 above)
Comment (vijikarthi) [inline gradle.properties]: Flink connector is configured to use 1.3.1 version and that's why I have changed the samples dependency to 1.3.1 but reverted the change now. Either way it should not create any issue.

File: sanity-samples/src/main/java/io/pravega/samples/flink/StreamUtils.java (hunk 2 above)
Comment (vijikarthi) [inline sanity-samples/src/main/java/io/pravega/samples/flink/StreamUtils.java]: Fixed.

File: sanity-samples/src/main/java/io/pravega/samples/flink/testruns/ExactlyOnceWriterTest.java (hunk 3 above)
Comment (vijikarthi) [inline sanity-samples/src/main/java/io/pravega/samples/flink/testruns/ExactlyOnceWriterTest.java]: Fixed

**Comment by vijikarthi** [comment]
Can we get this through if the feedbacks are addressed?
